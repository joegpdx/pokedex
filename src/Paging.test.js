import React from 'react';
import renderer from 'react-test-renderer';
import Paging from './Paging.js';

test('renders PokeItem.js correctly', () => {
    const tree = renderer
        .create(<Paging />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});