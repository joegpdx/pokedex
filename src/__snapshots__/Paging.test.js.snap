// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders PokeItem.js correctly 1`] = `
<p
  className="paging"
>
  No results, try another search
</p>
`;
